### DISCUSSION TITLE
Configuração de E-mail e Data Incorreta

### DISCUSSION THREAD

**Discussion by nakoniev** [RamonSilva20/mapos#1787] [Q&A]
Olá, estou tendo dificuldades em configurar o SMTP para envio de e-mails, já fiz de tudo que está ao meu alcance, inclusive testei a configuração em outro cliente de e-mail e o mesmo está funcionando normalmente, porém no Map-OS não funciona, nenhum ajuste nas configurações de e-mail funciona. Nos logs aparece a seguinte mensagem:

`ERROR - 2022-05-01 02:06:16 --> 220 ESMTP smtp.hostinger.com
<br /><pre>hello: 250-smtp.hostinger.com
250-PIPELINING
250-SIZE 48811212
250-ETRN
250-STARTTLS
250-AUTH PLAIN LOGIN
250-ENHANCEDSTATUSCODES
250-8BITMIME
250 DSN
</pre><pre>starttls: 220 2.0.0 Ready to start TLS
</pre><pre>hello: 250-smtp.hostinger.com
250-PIPELINING
250-SIZE 48811212
250-ETRN
250-AUTH PLAIN LOGIN
250-ENHANCEDSTATUSCODES
250-8BITMIME
250 DSN
</pre>Failed to authenticate password. Error: 535 5.7.8 Error: authentication failed: UGFzc3dvcmQ6`

Outra coisa, eu testei esses e-mails entre as 22:00 e 23:00 de 30/04/2022, e a data tanto no log quanto na lista de envios de e-mail aparece bem a frente, gostaria de saber como arrumar.

Desde já agradeço!

**Comment by nakoniev** [comment]
Problema resolvido. Era bloqueio na hospedagem. E da data era só o timezone do PHP.

**Reply by evertonrj** [reply]
Mano, boa noite.

Também uso a hospedagem da hostinger, consegue dar um help?
